stdlib/templates/README.adoc

```diff
@@ -1,3 +1,3 @@
 These templates are fragments of OCaml source files, which tools/unlabel uses
```

Comment:
tools/unlabel ⇒ tools/sync_stdlib_docs
(this is the last remaining instance of the unlabel name).